scripts/docs.js

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+'use strict';
+/* eslint-disable no-sync, no-console */
+
+const CodeGen = require('swagger-js-codegen').CodeGen;
+const fs = require('fs');
+const path = require('path');
+const zlib = require('zlib');
+
+const mustacheLambdas = {
+  /**
+   * Remove all non-alphanumeric characters from rendered text.
+   * @returns {funtion} function
+   */
+  alphanumeric: function () {
+    return function (text, render) {
+      return render(text).replace(/\W/g, '');
+    };
+  },
+
+  /**
+   * Replace newline characters in rendered text with <br/>s.
+   * @returns {funtion} function
+   */
+  markdownBreaks: function () {
+    return function (text, render) {
+      return render(text).replace(/\n/g, '<br/>');
```

Comment:
Fo we need to support CRLF for Windows?

Reply:
It might be worth calling out these line endings specifically in `.gitattributes`